catweasl.c

```diff
@@ -187,6 +188,7 @@ unsigned char Mk3ControlBit[] = {
 #define MEMSIZE 131072
 #define CREG(c) (c->private[0])
 #define PTR(c) (c->private[1])
+#define SECTEND(c) (c->private[2])
```

Comment:
I found `SECEND()` confusing not realizing at first that it implied a transitory state.  Would there be a better name for it to imply that nature, maybe `LASTSECEND()`?